### PR TITLE
Week 4 - lesson A - Intro to objects

### PR DESCRIPTION
**Pull request by rarmatei**

### CODE REVIEW

**Comment by rarmatei** [comment]
Makes sense, I've separated them

File: week-4/A-objects-intro/README.md [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
-
-- introduce objects as a concept
-- when can we use objects (to store what type of information)
-- the very specific object syntax (the commas after each property, the colon between property name and value, the curly brackets)
-- good practices for naming giving names to properties (not having spaces, what to do when there is a space)
-- the language to use when talking about an objects: object literal, properties, property names, property values
-- very simple exercises to get them used to declaring object literals
+### Objects
+
+When you think about real world objects, they have a bunch of properties that describe how its unique. Your laptop, for example, has a brand (Lenovo/Apple etc.), a screen size (13/15 inch), RAM (8/16GB) etc.
```
Comment (djgrant) [inline week-4/A-objects-intro/README.md]: One of the aspirations of the repo is to write the lessons in [Simple English](https://simple.wikipedia.org/wiki/Wikipedia:Simple_English_Wikipedia#Simple_English) (short sentences, limited vocab, simple grammar etc.)

The lesson is well written and understandable but we can make some of the sentences simpler. For example:

> Objects in the real world have properties that describe how they are unique.

File: week-4/A-objects-intro/README.md [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+
+As arrays in JavaScript are useful for describing a collection of things, objects are very useful for grouping together a bunch of characteristics to describe a specific entity.
```
Comment (djgrant) [inline week-4/A-objects-intro/README.md]: "Entity" is a technical word/jargon that I don't think adds anything here

Comment (djgrant) [reply week-4/A-objects-intro/README.md]: I'm not sure that the likening to arrays clarifies the description of an object

File: week-4/A-objects-intro/README.md [hunk 3]
```diff
@@ -1,8 +1,23 @@
...
+
+```js
+var laptop = {
+    brand: "Lenovo",
+    screenSize: 13,
+    isTouchscreen: true
+};
+```
+
+What's happening above is that we're **declaring a variable called laptop** and assigning to it...an **object literal**!
```
Comment (djgrant) [inline week-4/A-objects-intro/README.md]: > In the code above we are...

File: week-4/A-objects-intro/README.md [hunk 4]
```diff
@@ -1,8 +1,23 @@
...
+var laptop = {
+    brand: "Lenovo",
+    screenSize: 13,
+    isTouchscreen: true
+};
+```
+
+What's happening above is that we're **declaring a variable called laptop** and assigning to it...an **object literal**! 
+
+An object literal is composed of a set of curly bracket `{...}`, and in between a set of properties that describe that object: `brand`, `screenSize`, `isTouchScreen`.
```
Comment (djgrant) [inline week-4/A-objects-intro/README.md]: I've avoided describing syntax in the other lessons. I think the best description of the syntax is an example of the code itself.

Comment (djgrant) [reply week-4/A-objects-intro/README.md]: It's a short lesson without the description, but I think that's totally fine! Students seem to adverse to reading long descriptions.

Comment (rarmatei) [reply week-4/A-objects-intro/README.md]: You're right. I thought that this being such a simple lesson, just an intro on objects, I could at least focus on the syntax and the language to use.

However, there's no real need to write that down.

File: week-4/A-objects-intro/README.md (hunk 2 above)
Comment (rarmatei) [reply week-4/A-objects-intro/README.md]: Yep - good point

File: week-4/A-objects-intro/README.md (hunk 1 above)
Comment (rarmatei) [reply week-4/A-objects-intro/README.md]: Good comments! Please keep them coming if you see anything else, especially the "language/use of english" ones!

**Comment by rarmatei** [comment]
@djgrant I've made the changes. I thought I could still a section on "useful terms" when talking about objects. Don't know if the description of each term is too much?

File: week-4/A-objects-intro/README.md [hunk 5]
```diff
@@ -1,8 +1,18 @@
...
+
+```js
+var laptop = {
+    brand: "Lenovo",
+    screenSize: 13,
+    isTouchscreen: true
+};
+```
+
+Useful words to remember when talking about objects:
```
Comment (djgrant) [inline week-4/A-objects-intro/README.md]: Great idea

File: week-4/A-objects-intro/exercise-part-1.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+/*
+
+Think of 5 different real world "things" that you 
+can describe with a JavaScript object
```
Comment (djgrant) [inline week-4/A-objects-intro/exercise-part-1.js]: "Describe using code" instead of "think"?

**Pull request merged**